cms/admin/placeholderadmin.py

```diff
@@ -1105,6 +1106,7 @@ def delete_plugin(self, request, plugin_id):
             "protected": protected,
             "opts": opts,
             "app_label": opts.app_label,
+            "media": Media(js=['admin/js/vendor/jquery/jquery.min.js', 'admin/js/jquery.init.js'])
```

Comment:
'admin/js/vendor/jquery/jquery.min.js' is redundant when 'admin/js/jquery.init.js' is present because jquery.init.js is already dependant on jquery.

Reply:
Removing `'admin/js/vendor/jquery/jquery.min.js'` as you suggested isn't such a good idea. If I do that, I get this error in the browser console:

![Screenshot 2020-10-15 at 13 51 05](https://user-images.githubusercontent.com/1132472/96119912-f8d9e680-0eed-11eb-931a-5ceeffd26850.png)

Reply:
Strange, the Django docs mention that you only need to add the init: https://docs.djangoproject.com/en/3.1/releases/2.2/#merging-of-form-media-assets

Reply:
Anyway, adding that `media`-attribute, shouldn't do any harm. Did you try yourself without `djangocms_admin_style` in `INSTALLED_APPS`?